test/test_datahandling.py

```diff
@@ -83,6 +84,9 @@ def setUp(self):
         m = np.reshape(np.arange(60), (4, 5, 3))
         self.f3d = dh.Field(m)
 
+        x, y = np.meshgrid(np.linspace(0,1,100), np.linspace(0,1,100), indexing='ij', sparse=True)
+        self.f2d_fine = dh.Field(np.sin(x)*np.cos(y))
```

Comment:
maybe its easier to sample the function between 0 and 2pi or 4pi and then you know, that the integration tests should yield zero.